docs/Manual/03 Schedules.md

```diff
@@ -16,39 +16,40 @@ for i in range(16):
         for k in range(11):
             C[i, j] += A[i, k] * B[k, j]
 ```
-In other words, each of the logical pseudo-code loops in `nest` becomes an actual for-loop in the default schedule.
+In other words, each of the logical pseudo-code loop in `nest` become an actual for-loop in the default schedule. 
```

Comment:
I think "each of the logical pseudo-code loops ... becomes" is more grammatically correct.